satpy/tests/reader_tests/modis_tests/_modis_fixtures.py

```diff
@@ -326,8 +339,35 @@ def _create_struct_metadata(geo_resolution: int) -> str:
     return struct_metadata_header
 
 
+def _create_struct_metadata_cmg(res) -> str:
+    # Case of a MOD09 file
+    gridline = 'GridName="MOD09CMG"\n'
+    upleft = "UpperLeftPointMtrs=(-180000000.000000,90000000.000000)\n"
+    upright = "LowerRightMtrs=(180000000.000000,-90000000.000000)\n"
+    if res == -9999:
+        # Case of a MCD43 file
+        gridline = 'GridName="MCD_CMG_BRDF_0.05Deg"\n'
+        upleft = "UpperLeftPointMtrs=(-180.000000,90.000000)\n"
+        upright = "LowerRightMtrs=(180.000000,-90.000000)\n"
+
+    struct_metadata_header = ("GROUP=SwathStructure\n"
+                              "END_GROUP=SwathStructure\n"
+                              "GROUP=GridStructure\n"
+                              "GROUP=GRID_1\n"
+                              f"{gridline}\n"
+                              "XDim=7200\n"
+                              "YDim=3600\n"
+                              f"{upleft}\n"
+                              f"{upright}\n"
+                              "END_GROUP=GRID_1\n"
+                              "END_GROUP=GridStructure\nEND")
+    return struct_metadata_header
+
+
 def _create_header_metadata() -> str:
-    archive_metadata_header = "GROUP = ARCHIVEDMETADATA\nEND_GROUP = ARCHIVEDMETADATA\nEND"
+    archive_metadata_header = ("GROUP = ARCHIVEDMETADATA\n"
+                               'TEST_URL = "http://modis.gsfc.nasa.gov/?some_val=100"\n'
```

Comment:
Does this work? For consistency of quote type.

```suggestion
                               "TEST_URL = \"http://modis.gsfc.nasa.gov/?some_val=100\"\n"
```

Reply:
Ruff complains if I do this.